### PR TITLE
Sxg chapter03

### PR DESCRIPTION
**Pull request by sebing**

### CODE REVIEW

File: chapter03/sxg/ex_3_2.pl [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!perl
+use warnings;
+
+@names = qw( fred betty barney dino wilma pebbles bamm-bamm );
+
+print "Enter few numbers between 1 and " . ($#names+1) . ":\n";
```
Comment (thatguy59) [inline chapter03/sxg/ex_3_2.pl:6]: Nice way to keep it maintainable

**Comment by alexdeas** [comment]
It looks like you've been merging master into your branch, rather than creating new branches from master. It's probably easier create your branches each week from master, and might avoid some branch inconsistencies later on.

File: chapter03/sxg/ex_3_2.pl [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+chomp(@numbers = <STDIN>);
+
+print "You chose:\n";
+foreach (@numbers) {
+	if($_>= 1 && $_ <= $#names+1){
+		print "$names[ $_ - 1 ]\n";	
+	} else {
+		print "Opps! You are not listening to me. You enter: $_\n";
+	}
+
```
Comment (JPeace2) [inline chapter03/sxg/ex_3_2.pl:17]: Be careful using $_ as it does get overwritten at various points in a program.

Comment (qianliok) [inline chapter03/sxg/ex_3_2.pl:17]: - number range validation is good
- I prefer `foreach $num ( @numbers )`  over `&_`

File: chapter03/sxg/ex_3_3.pl [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!perl
+use warnings;
+
+print "Enter text:\n"; 
+
+chomp(@lines = <STDIN>);
+@sorted = sort @lines;
```
Comment (thatguy59) [inline chapter03/sxg/ex_3_3.pl:7]: You can also reuse lines here, instead of also having sorted

**Pull request merged**